extensions/wikia/Flags/controllers/FlagsController.class.php

```diff
@@ -73,11 +97,11 @@ public function getFlagsForPageWikitext( $pageId ) {
 					);
 				}
 
-				$flagsWikitext = $flagView->wrapAllFlags( $templatesCalls );
+				return $flagView->renderFlags( $templatesCalls, $pageId );
```

Comment:
Since you are using `wfProfileIn` you should call `wfProfileOut` on all returns, but we don't really rely on `wfProfile*` for profiling anymore anyway IIRC.